.goreleaser.yaml

```diff
@@ -0,0 +1,39 @@
+before:
+  hooks:
+    - go mod tidy
+    - go generate ./...
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
```

Comment:
What's this for?

Reply:
I used this when i was building locally ... It helps skip validation and keeps everything simple and local... i could remove that.. doesn't affect anything

[snapshot](https://goreleaser.com/customization/snapshots/)

Reply:
Yeah comment it out